hpx/components/process/util/posix/initializers/run_exe.hpp

```diff
@@ -0,0 +1,93 @@
+// Copyright (c) 2006, 2007 Julio M. Merino Vidal
+// Copyright (c) 2008 Ilya Sokolov, Boris Schaeling
+// Copyright (c) 2009 Boris Schaeling
+// Copyright (c) 2010 Felipe Tanus, Boris Schaeling
+// Copyright (c) 2011, 2012 Jeff Flinn, Boris Schaeling
+//
+// Distributed under the Boost Software License, Version 1.0. (See accompanying
+// file LICENSE_1_0.txt or copy at http://www.boost.org/LICENSE_1_0.txt)
+
+#ifndef HPX_PROCESS_POSIX_INITIALIZERS_RUN_EXE_HPP
+#define HPX_PROCESS_POSIX_INITIALIZERS_RUN_EXE_HPP
+
+#include <hpx/config.hpp>
+
+#if !defined(HPX_WINDOWS)
+#include <hpx/runtime/serialization/string.hpp>
+#include <hpx/components/process/util/posix/initializers/initializer_base.hpp>
+
+#include <boost/filesystem.hpp>
+#include <boost/shared_array.hpp>
+#include <string>
+
+namespace hpx { namespace components { namespace process { namespace posix {
+
+namespace initializers {
+
+class run_exe_ : public initializer_base
+{
+public:
+    run_exe_()
+    {
+    }
+
+    explicit run_exe_(const std::string &s) : s_(s), cmd_line_(new char*[2])
```

Comment:
Don't ask, this was taken directly from Boost.Process. A static array would do just fine, I guess.